en/stripe-2.md

```diff
@@ -0,0 +1,261 @@
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/03fdaee9d13b521b773e7aed365d83e0.png)](https://diverta.gyazo.com/03fdaee9d13b521b773e7aed365d83e0)
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/cc0926ce21366db86a5a545014dfe375.png)](https://diverta.gyazo.com/cc0926ce21366db86a5a545014dfe375)
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/13a558163253ed7b19acf9805a2898a6.png)](https://diverta.gyazo.com/13a558163253ed7b19acf9805a2898a6)
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/e00d97f4fa571111e43c869924f2eba5.png)](https://diverta.gyazo.com/e00d97f4fa571111e43c869924f2eba5)
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/0029537b14a95543876fc455a60b3a91.png)](https://diverta.gyazo.com/0029537b14a95543876fc455a60b3a91)
+-->
+
+To check if your billing process is working as expected, verify the endpoint response from the Swagger UI and make a test payment.
+
+1. From the sidebar menu, select the API containing your checkout endpoint. On the endpoint list screen, click [Swagger UI].
+2. On the Swagger UI screen, select your checkout endpoint and click [Try it out].
+3. Scroll down and click [Execute].
+4. Copy the URL in the response body.
+
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/bd0480d5b7ba9a21dd703119387b99ac.jpg)](https://diverta.gyazo.com/bd0480d5b7ba9a21dd703119387b99ac)
+
+5. Access the URL in your browser. You should see the payment screen below. Enter your payment details and click [Subscribe].
+
+[[info]] You can use [test cards](https://stripe.com/docs/testing#international-cards) to simulate various scenarios.  
+
+6. If the payment was successful, you will be redirected to the payment success page you set in your endpoint.
+
+<!--
+Stripe-side verification steps needed
+-->
+
+Once completed, the member is automatically added to the member group(s) for the product(s). They will also receive a Stripe member ID and subscription ID attached to their Kuroco account. For more details, see [Second tutorial - Customer ID & Subscription ID](#link).
+
+
+### Verifying the subscription cancellation
+
+<!--
+To test the cancellation process, go to the Swagger UI and click [Try it out] for your cancellation endpoint. 
+
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/8b00ca3d85afb619f21e465a4c58f176.png)](https://diverta.gyazo.com/8b00ca3d85afb619f21e465a4c58f176)
+
+Scroll down and click [Execute].
+
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/e9e79640827b9df7295d00da4803c7f7.png)](https://diverta.gyazo.com/e9e79640827b9df7295d00da4803c7f7)
+-->
+
+To test the cancellation process, repeat checkout verification steps 1 - 3 for the cancellation endpoint. The member will be automatically removed from the associated product groups, and their Stripe subscription ID will be unlinked from their Kuroco account. For more details, see [Second tutorial - Customer ID & Subscription ID](#link).
```

Comment:
Another note, we can skip the screenshots for this, but add a line to mention to check their Stripe account and Kuroco has been updated accordingly.
- Stripe should remove the subscription from the user
- Kuroco should remove the member group from the user (the info about customer_id and subscription_id could be excluded for this tutorial since it's not relevant, and it should only be expanded upon in the 2nd doc)